resources/assets/components/PagePlaceholder.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const PagePlaceholder = () => (
+  <div className="placeholder flex-center-xy">
+    <div className="spinner" />
```

Comment:
For a future PR I'd love to see Tongue Cat incorporated here somehow 😝

Reply:
yessss 👅